src/solver/optimisation/opt_appel_solveur_lineaire.cpp

```diff
@@ -97,15 +97,15 @@ struct SimplexResult
     mpsWriterFactory mps_writer_factory;
 };
 
-static SimplexResult OPT_TryToCallSimplex(
-        const OptimizationOptions& options,
-        PROBLEME_HEBDO* problemeHebdo,
-        Optimization::PROBLEME_SIMPLEXE_NOMME& Probleme,
-        const int NumIntervalle,
-        const int optimizationNumber,
-        const OptPeriodStringGenerator& optPeriodStringGenerator,
-        bool PremierPassage,
-        IResultWriter& writer)
+static SimplexResult OPT_TryToCallSimplex(const OptimizationOptions& options,
+                                          PROBLEME_HEBDO* problemeHebdo,
+                                          Optimization::PROBLEME_SIMPLEXE_NOMME& Probleme,
+                                          const int NumIntervalle,
+                                          const int optimizationNumber,
+                                          const OptPeriodStringGenerator& optPeriodStringGenerator,
+                                          bool PremierPassage,
+                                          IResultWriter& writer,
+                                          OrtoolsLogHandler& ortools_logger)
```

Comment:
Use `IResultWriter` to write anything, Antares_Simulator doesn't write file directly.

This is because sometimes we use zip (option `-z`)

Idea: keep a `std::string` inside `ortools_logger`, then write it to IResultWriter in the destructor.

Reply:
will do so

Reply:
done